Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -208,6 +206,21 @@ void GameFile::SaveCache()
   // TODO
 }
 
+QString GameFile::GetFileName() const
+{
+  return QFileInfo(m_path).fileName();
```

Comment:
shouldn't `<QDir>`, `<QFile>` and `<QFileInfo>` be included in the cpp file as well?